rokka/sourceimages.go

```diff
@@ -36,6 +39,11 @@ type GetSourceImageResponse struct {
 	UserMetadata map[string]interface{} `json:"user_metadata"`
 }
 
+type CreateSourceImageResponse struct {
+	Total string                   `json:"total"`
```

Comment:
@chregu fyi, the response after uploading a source image returns the total as a string. usually this is an integer (e.g. list source images)